setup/github-bot/ansible-playbook.yaml

```diff
@@ -0,0 +1,59 @@
+---
+- hosts: github-bot
+
+  remote_user: root
+
+  tasks:
+
+    - include_vars: ansible-vars.yaml
+      tags: vars
+
+    - name: General | Add the NodeSource Node.js repo
+      command: "bash -c 'curl -sL https://deb.nodesource.com/setup_6.x | bash -'"
+      tags: general
+
+    - name: General | APT Update and upgrade
+      apt: update_cache=yes upgrade=full
+      tags: general
+
+    - name: General | Install baseline packages
+      apt: name={{ item }} state=latest
+      with_items: "{{ baseline_packages }}"
+      tags: general
+
+    - name: General | Install required packages
+      apt: name={{ item }} update_cache=yes state=latest
+      with_items: "{{ packages }}"
+      tags: general
+
```

Comment:
oh, yeah -- you probably want that as part of bootstrapping then. See for instance [this](https://github.com/nodejs/build/pull/446/files#diff-54a20c9aff06229996d3d614d66b5425R4) where we run raw commands to avoid calling python/apt.